extensions/wikia/ArticleVideo/ArticleVideoContext.class.php

```diff
@@ -70,12 +70,36 @@ public static function getFeaturedVideoData( string $prefixedDbKey ) {
 			$videoData['dfpContentSourceId'] = $wg->AdDriverDfpOoyalaContentSourceId;
 			$videoData['metadata'] = self::getVideoMetaData( $videoData );
 
+			$videoData = self::getVideoDataWithAttribution($videoData);
+
 			return $videoData;
 		}
 
 		return [];
 	}
 
+	private static function getVideoDataWithAttribution( $videoData ) {
+		if ( empty( $videoData['playlist'] ) || empty( $videoData['playlist'][0] ) ) {
+			return $videoData;
+		}
+
+		$playlist = $videoData['playlist'][0];
```

Comment:
this variable name is misleading, it does not contain playlist but single video. Can we rename it?